src/status_im/contexts/settings/wallet/keypairs_and_accounts/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.view
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.constants :as constants]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.actions.view :as actions]
+            [status-im.contexts.settings.wallet.keypairs-and-accounts.style :as style]
+            [utils.address :as utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [customization-color emoji name address]}]
+              {:account-props {:customization-color customization-color
+                               :size                32
+                               :emoji               emoji
+                               :type                :default
+                               :name                name
+                               :address             address}
+               :networks      []
+               :state         :default
+               :action        :none}))))
```

Comment:
This code is mostly the same as this function here: https://github.com/status-im/status-mobile/blob/b907881d59d97f34b8dde16f24ed6003f16d16b5/src/status_im/contexts/wallet/add_account/create_account/select_keypair/view.cljs#L32-L48

But inside wallet setting we do not want to show the network shorthands. We should probably find a place to share this code, thoughts?